### ISSUE TITLE
PHP message: PHP Warning:  Undefined array key "hid" in /dokuwiki-test/inc/html.php on line 114

### ISSUE THREAD

**Issue by fiwswe** [splitbrain/dokuwiki#3751]
My setup:
OpenBSD 7.1 release, all patches applied, amd64, [`httpd(8)`](https://man.openbsd.org/OpenBSD-7.1/amd64/httpd.8), PHP-FPM 8.0.22, HTTP Basic authentication configured, DokuWiki 2022-07-31 "Igor".
The Wiki is a copy of an existing Wiki based on Hogfather upgraded to Igor according to the instructions (https://www.dokuwiki.org/install:upgrade).

Thanks!
fiwswe

**Comment by fiwswe** [comment]
See also: #3728

**Comment by Klap-in** [comment]
Do have a trace of this error, or could you make one? Such that we can fiddle out where is originates? And reproduce it.
(The development version of the wiki logs these warnings with more info…)

**Comment by fiwswe** [comment]
I'll have to look into this as I currently have no idea of how to get/enable/whatever the _development version_ ;-) Or where it dumps its logs.

If you have a quick pointer to some instructions, I'll see what I can do. If not I'll research this…

Thx.
fiwswe

**Comment by mprins** [comment]
log files end up in `/var/log/` there should be a php log there unless you reconfigured that to go elsewhere

**Comment by splitbrain** [comment]
Logs are in data/log/error/* and there's a log viewer in the admin interface

**Comment by fiwswe** [comment]
Cool, I hadn't noticed the integrated log viewer yet.

At the moment I can't reproduce the error. I'll try to get it to reproduce and will update this issue when I do.

Just to recap the procedure:
- I activated `allowdebug` in the settings (not sure if this is required?)
- I am looking at logs produced by PHP-FPM (`/var/www/logs/error.log` this is where I first noticed the warnings, `/var/log/php80-fpm.log`) and DokuWiki `…/data/cache/debug.log`)

Anything else I should be doing to get maximum info when the problem resurfaces?

Thx.
fiwswe
